examples/5-nextjs/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "scaffold-test",
+  "scripts": {
+    "install-mst-gql": "bash ../../scripts/install-mst-gql.sh",
+    "scaffold": "../../generator/mst-gql-scaffold.js --roots 'Todo' --excludes 'CacheControlScope,Query,Subscription' --outDir src/models/ --format ts http://localhost:4000/graphql",
+    "build": "tsc",
+    "dev": "next",
+    "start": "next start"
+  },
+  "dependencies": {
+    "apollo-server-express": "^2.6.1",
+    "mobx": "^5.9.4",
+    "mobx-react": "^6.0.3",
+    "mobx-state-tree": "^3.14.0",
+    "mst-gql": "file:./my-package1559643209.tgz",
```

Comment:
could you please remove this line? mst-gql will be installed via install-mst-gql.sh scripts and yarn throws error when doing yarn install if this file dependency is present